HandsOnTutorial/400-using-cloud-channels.fsx

```diff
@@ -0,0 +1,51 @@
+﻿(*** hide ***)
+#load "Thespian.fsx"
+#load "Azure.fsx"
+
+open System
+open System.IO
+open MBrace.Core
+open MBrace.Azure
+open MBrace.Flow
+
+// Initialize client object to an MBrace cluster:
+let cluster = 
+//    getAzureClient() // comment out to use an MBrace.Azure cluster; don't forget to set the proper connection strings in Azure.fsx
+    initThespianCluster(4) // use a local cluster based on MBrace.Thespian; configuration can be adjusted using Thespian.fsx
+
+(**
+# Using Cloud Queues
+
+This tutorial illustrates creating and using cloud channels, which allow you to send messages between
+cloud workflows.
+ 
+Before running, edit credentials.fsx to enter your connection strings.
+*)
+
+(** Create an anonymous cloud channel: *) 
+let queue = CloudQueue.New<string>() |> cluster.RunOnCloud
+
+(** Send to the channel by scheduling a cloud process to do the send: *)
+CloudQueue.Enqueue (queue, "hello") |> cluster.RunOnCloud
```

Comment:
Why don't you use instance notation?  Using static methods just doesn't give clean code for these kind of abstractions.